src/components/Patient/PatientHome.tsx

```diff
@@ -247,21 +245,6 @@ export const PatientHome = (props: {
                 </div>
               </div>
             </div>
-            <div className="py-2">
-              {patientData.death_datetime && (
-                <div>
-                  <Button
-                    id="death-report"
-                    className="my-2 w-full"
-                    name="death_report"
-                    onClick={() => navigate(`/death_report/${id}`)}
-                  >
-                    <CareIcon icon="l-file-download" className="text-lg" />
-                    {t("death_report")}
```

Comment:
these translations can be removed as well as they no longer exist in the codebase